assets/js/components/wp-dashboard/WPDashboardPopularPages.js

```diff
@@ -96,14 +99,18 @@ export default function WPDashboardPopularPages( props ) {
 		return data;
 	} );
 
-	if ( loading ) {
-		return <PreviewTable rows={ 6 } />;
+	if ( ! isGatheringData ) {
+		return null;
 	}
```

Comment:
This isn't really accurate - `isGatheringData` could be `undefined`, in which case this should be in a loading state I think?

Reply:
👍